setup.cfg

```diff
@@ -2,11 +2,8 @@
 addopts = -rsxX --reuse-db --tb=native
 norecursedirs = .git docs bin staticfiles
 testpaths = standup
-python_files=test*.py
-DJANGO_SETTINGS_MODULE=standup.settings
-SECRET_KEY=foo
```

Comment:
These were ignored because we're not using a pytest plugin that puts these in the environment. So I nixed them.